core/game_object.lua

```diff
@@ -2419,165 +2423,237 @@ function loadAPIs()
+            G.PALETTE = {
+                old_colours = {HEX("235955"),HEX("3c4368"),HEX("f06b3f"),HEX("f03464")},
+                new_colours = {HEX(SMODS.AltTextures[self.type][self.name].suits.Clubs), HEX(SMODS.AltTextures[self.type][self.name].suits.Spades), HEX(SMODS.AltTextures[self.type][self.name].suits.Diamonds), HEX(SMODS.AltTextures[self.type][self.name].suits.Hearts)}
+            }
+            G.ASSET_ATLAS["ui_1"][self.name].image_data:mapPixel(G.FUNCS.recolour_image)
+            G.ASSET_ATLAS["ui_1"][self.name].image = love.graphics.newImage(G.ASSET_ATLAS["ui_1"][self.name].image_data, {mipmaps = true, dpiscale = G.SETTINGS.GRAPHICS.texture_scaling})
+        elseif self.type == "Spectral" then
+            atlas_key = "Spectral"
+            if G.ASSET_ATLAS[self.atlas_key.."_e"] then
+                G.ASSET_ATLAS["soul"][self.name] = G.ASSET_ATLAS[self.atlas_key.."_e"]
+            else
+                G.ASSET_ATLAS["soul"][self.name] = {image_data = G.ASSET_ATLAS["soul"].image_data:clone()}
+                G.ASSET_ATLAS["soul"][self.name].image = love.graphics.newImage(G.ASSET_ATLAS["soul"][self.name].image_data, {mipmaps = true, dpiscale = G.SETTINGS.GRAPHICS.texture_scaling})
+            end
+        elseif self.type == "Seal" then
+            atlas_key = "centers"
+        elseif self.type == "Tag" then
+            atlas_key = "tags"
+        else
+            atlas_key = self.type
         end
-        return { HEX(base_colour) }
+        G.ASSET_ATLAS[atlas_key][self.name] = G.ASSET_ATLAS[self.atlas_key]
     end
 
-    for k, v in pairs(G.P_CENTER_POOLS.Tarot) do
-        SMODS.Consumable:take_ownership(v.key, { atlas = "Tarot" })
+    -- Call the `generate_colours` function of the appropriate type
+    function SMODS.AltTexture:create_colours(type, base_colours)
+        if SMODS.ConsumableTypes[type].generate_colours then
+            return SMODS.ConsumableTypes[type]:generate_colours(base_colours)
+        end
+        return {(type(base_colours) == 'table' and HEX(base_colours[1]) or HEX(base_colours))}
+    end 
+
+    -- Default palettes defined for base game suits
+    SMODS.AltTexture({
+        key = "base_cards",
+        old_colours = {},
+        new_colours = {},
+        suits = {
+            Clubs = "235955",
+            Spades = "3c4368",
+            Diamonds = "f06b3f",
+            Hearts = "f03464"
+        },
+        type = "Suit",
+        name = "Default",
+        palette = true,
+    })   
+    SMODS.AltTexture({
+        key = "high_contrast_cards",
+        texture = true,
+        atlas_key = "cards_2",
+        suits = {
+            Clubs = "008ee6",
+            Spades = "3c4368",
+            Diamonds = "e29000",
+            Hearts = "f83b2f"
+        },
+        type = "Suit",
+        name = "High Contrast",
+    })
+    for k,v in pairs(G.P_CENTER_POOLS.Tarot) do
```

Comment:
These `take_ownership` calls flood logs at startup for no good reason. Perhaps they should be made silent?